students/brandon_nguyen/session03/mailroom.py

```diff
@@ -0,0 +1,114 @@
+import sys
+import unittest
+from decimal import Decimal
+
+#!/usr/bin/env python3
+#Week3 Excercise mailroom part 1
+#Student: Brandon Nguyen - Au2018
+
+#global define data structure
+donor_db = [("Brandon Nguyen", [4500.33,350.87,300.05]),
+          ("Shawn DeGraw", [5500.00, 345.05,571.33]),
+          ("Jacqueline Lee",[4300.00,3200.00,230.13]),
+          ("Aidan Nguyen", [4300.00,200.00,238.23]),
+          ]
+promptTxt = "\n".join(("Welcome to the mailroom program!",
+          "Please choose from below 3 options:",
+          "1 - Send a Thank You",
+          "2 - Create a Report",
+          "3 - Quit",
+          ">>> "))
+#The script should prompt the user (you) to choose from a menu of 3 actions: “Send a Thank You”, “Create a Report” or “quit”.
+
+#think about a main menu function that can be called any where.
+def main_menu():
+    promptTxt = "\n".join(("Welcome to the mailroom program!",
+          "Please choose from below 3 options:",
+          "1 - Send a Thank You",
+          "2 - Create a Report",
+          "3 - Quit",
+          ">>> "))
```

Comment:
Not sure why you need this function. You are only redefining the `promptTxt` variable with the same content.

In fact, I don't see this function called from anywhere (from a cursory look, so maybe I'm wrong), so this function better be just removed.

Reply:
I was thinking of something but never removed it.